velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -25,6 +25,7 @@ namespace facebook::velox::substrait {
 /// This class is used to convert the Substrait plan into Velox plan.
 class SubstraitVeloxPlanConverter {
  public:
+  SubstraitVeloxPlanConverter(memory::MemoryPool* pool) : pool_(pool) {}
```

Comment:
This is a nice refactor. Would it be possible to extract it into a separate PR to make it easier to review this PR?

Reply:
Ok, will do it.